resources/toml-check.py

```diff
@@ -41,6 +41,12 @@ def error(msg):
             f'must be set to "free-shared" or "standard"',
         )
 
+    if 'lesson_areas' not in meta:
```

Comment:
We should probably check if it's a list, `isinstance(meta['lesson_areas'],list)`